entrypoint.sh

```diff
@@ -5,13 +5,17 @@ set -e
 # setup the environment
 if [ "$1" = 'SickBeard.py' ]; then
   if [ ! -f ${APP_DATA}/config.ini ]; then
-    cp /template/config.ini ${APP_DATA}/
+    exec gosu ${APP_UID:=0}:${APP_GID:=0} cp /template/config.ini ${APP_DATA}/
+  fi
+  CFG_DIR_UID=`ls -lnd "$APP_DATA" | awk 'NR==1 {print $3}'`
+  CFG_DIR_GID=`ls -lnd "$APP_DATA" | awk 'NR==1 {print $4}'`
+  if [ $APP_UID -ne $CFG_DIR_UID ] && [ $APP_GID -ne $CFG_DIR_GID ]; then
```

Comment:
From what I could find, using ls was the most portable solution to determining file permissions. stat has variable output across platforms whereas `ls` does not.

We have to make a decision on portability vs. slightly different looking code. Your decision.